google/resource_compute_backend_bucket.go

```diff
@@ -16,40 +29,37 @@ func resourceComputeBackendBucket() *schema.Resource {
 		Delete: resourceComputeBackendBucketDelete,
 
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: resourceComputeBackendBucketImport,
 		},
 
 		Schema: map[string]*schema.Schema{
-			"name": &schema.Schema{
-				Type:         schema.TypeString,
-				Required:     true,
-				ForceNew:     true,
-				ValidateFunc: validateGCPName,
-			},
-
-			"bucket_name": &schema.Schema{
+			"bucket_name": {
 				Type:     schema.TypeString,
 				Required: true,
 			},
-
-			"description": &schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
```

Comment:
Shouldn't this be ForceNew?

Reply:
Ah yeah, that was broken by the same change that broke import- it'll be fixed once my update changes are in so I don't feel bad fixing it manually for now.